DiegoCelisT.html

```diff
@@ -0,0 +1,131 @@
+                </p>
+                <p>
+                Depois do que aconteceu no St Mary's Stadium, onde os liderados por <span>Ole Gunnar Solskjaer</span> tiveram que largar tudo para manter os três pontos, ninguém imaginou que os meninos voltariam a encontrar a vitória desta forma excepcional, selando um triunfo raramente visto na criação deste torneio.
+                </p>
+                <a href="https://www.manutd.com/es/news/detail/sp-match-analysis-from-man-utd-v-southampton-2-february-2021">Ler mais</a>
+            </div>
+            <div>
+                <img src="Images_DiegoCelisT/Noticia2.JPG" alt="Foto de Marcos Rojo no campo de treinamento" width="600" height="250">
+                <p>
+                Depois de sete anos no clube, Marcos Rojo voltou à Argentina para continuar sua carreira no Boca Juniors.
+                </p>
+                <p>
+                O Manchester United fechou um acordo com o último campeão argentino de futebol para selar o passe do zagueiro de 30 anos, que não jogava na seleção principal desde novembro de 2019.
+                </p>
+                <a href="https://www.manutd.com/es/news/detail/marcos-rojo-completes-permanent-move-from-man-utd-to-boca-juniors">Ler mais</a>
+            </div>
+        </article> 
+        
+        
+        <article id="Videos-recentes">
+            <h2>Videos Recentes</h2>
+            <nav>
+                <a href="https://www.youtube.com/watch?v=vU7jClGX1Pg">
+                    <img src="Images_DiegoCelisT/Últimos Videos 1.JPG" alt="Entrevista de Rashford depois do jogo contra Southampton" width="250" height="300">
+                </a>
+                <a href="https://www.youtube.com/watch?v=FDyHIhNIYJ0">
+                    <img src="Images_DiegoCelisT/Últimos Videos 2.JPG" alt="Reação de OGS depois do jogo contra Southampton" width="250" height="300">
+                </a>
+                <a href="https://www.youtube.com/watch?v=R6mmohKTYVY">
+                    <img src="Images_DiegoCelisT/Últimos Videos 3.JPG" alt="Melhores momentos do jogo contra Southampton" width="250" height="300">
+                </a>          
+            </nav>
+        </article>
+
+        <article id="Últimos-Jogos">
+            <h2>Últimos jogos</h2>
+            <img src="Images_DiegoCelisT/MU-SO.JPG" alt="Dados do Jogo Manchester United vs Southampton" width="250" height="250">
+            <img src="Images_DiegoCelisT/MU-EV.JPG" alt="Dados do Jogo Manchester United vs Everton" width="250" height="250">
+            <img src="Images_DiegoCelisT/MU-WH.JPG" alt="Dados do Jogo Manchester United vs West Ham" width="250" height="250">
+            <img src="Images_DiegoCelisT/WB-MU.JPG" alt="Dados do Jogo West Brom vs Manchester United" width="250" height="250">
+        </article>
+
+        <article id="Galerias">
+            <h2>Galerias</h2>
+            <div>
+                <picture>
```

Comment:
O elemento `figcaption` precisa ser utilizado em conjunto com o elemento `figure` especificamente, ao invés do `picture`.
```suggestion
                <figure>
```

Reply:
Sim!!!! confundí o comando, obrigado pela correção, vou consertar